mlflow/spark.py

```diff
@@ -39,6 +41,47 @@ def log_model(spark_model, artifact_path, conda_env=None, jars=None):
                      jars=jars, conda_env=conda_env)
 
 
+def _tmp_path(len):
+    return ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(len))
+
+
+class _DFS:
+    _fs = None
+
+    @classmethod
```

Comment:
Could we have a test for this class specifically? I know it's covered by test_spark, but there's a lot of complicated JVM-y things happening here that would be good to confirm work as expected by themselves.

Reply:
Mostly just want to test that all the `_jvm` stuff is working as expected.